lib/progress_bar.rb

```diff
@@ -6,9 +6,10 @@ class ProgressBar
   Error = Class.new(StandardError)
   ArgumentError = Class.new(Error)
 
-  attr_accessor :count, :max, :meters
+  attr_accessor :count, :max, :meters, :prefix, :suffix
+  attr_reader :terminal_width # Useful for prefix and suffix changes
```

Comment:
I felt that this value can be useful if someone is now adding prefixes and suffixes they may need this value in outside calculations